Rakefile

```
@@ -92,6 +92,21 @@ end
 namespace :unicode do
   ucd_dir = "./generated/ucd"
 
+  desc 'Rebuild Rust generated Rust sources from Unicode data'
+  task :build do
+    system("ucd-generate --version") || begin
+      puts 'Please install "ucd-generate" first with `cargo install ucd-generate`'
+      exit(1)
+    end
+
+    dest = "./generated/case_mapping.rs"
+    if system("ucd-generate case-mapping #{ucd_dir} --flat-table > #{dest}")
```

We only need `TITLE`, and the unused `UPPER` and `LOWER` is surprisingly hard to be ignored or autocorrected by `dead_code` rule. I'm trying to make it possible to generate only `TITLE` with a new `ucd-generate` flag (https://github.com/BurntSushi/ucd-generate/compare/master...choznerol:choznerol/cli-case-mapping-include-flag?expand=1)
```suggestion
if system("ucd-generate case-mapping #{ucd_dir} --flat-table --include TITLE > #{dest}")
```